test/fixtures/questions.yml

```diff
@@ -1,6 +1,7 @@
 # Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
 one:
   content: What is your gender?
+  status: draft
```

Comment:
This is not needed as ALL questions begin as drafts, and it may not be desirable in the context of the other tests.